packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,335 @@
+- Again, these are guesstimates. We’re not worried about the exact height of each card. Rather, we’reusing our column structure as a guide.
+
+
+After placing our prioritized and sized card list on the four-column grid, the result should look like this example:
+
+<img src="./img/dashboard-first-cut.png" alt="dashboard layout populated with placeholder cards" width="1680"/>
+
+Note how loosely we stuck to those sizes. The details card is pretty small, but the events and history cards ended up being more like a large card. That’s fine.
+
+When choosing whether or not your card should be wide or tall, it helps to consider the format of the information within the card. For example, the inventory and events cards both contain a list of data. Lists tend to lend themselves well to taller cards. The consumers card (bottom right) will contain a line graph. A line graph showing data over time lends itself well to a wider card.
+
+You can now iterate on the arrangement of your cards. Approach this arrangement like a puzzle. If you have a strong prioritization of cards, that should be your primary driver. You want the most important cards at the top, with less important cards at the bottom. It’s OK to change the size of your cards a bit to make them fit better. In our example, the events and inventory cards probably don’t need to be that tall. Maybe we can find a better use of that space for another card that needs more room.
+
+
+The only boundaries you’ll encounter in this step are the general sizes you defined earlier. Don’t limit yourself by assuming that the inventory card always has to be that tall. We don’t know exactly how tall it really has to be yet, we just have a general idea that it probably should be medium-ish. If you find yourself struggling with a certain column layout, try a different one to see if it solves any problems.
+
+You can rearrange cards multiple times until you find the layout that works:
+
+<img src="./img/dashboard-iterations.png" alt="dashboard layout iterations" width="5450"/>
+
+We knew details and performance were our top priority items, so we kept those up top and mainly iterated around the rest of the cards. Note how we changed the height of some of the cards to better fit the layout.
+
+Dynamic cards deserve special considerations. You want to avoid situations such as having a one column dynamic card expand and push down a two column card below it, creating a gap of unused space. We’ll explore how this might play out with ther events card in our next example.
+
+<img src="./img/dashboard-what-to-avoid.png" alt="dashboard layout with a gap" width="1680"/>
+
+We could solve this gap by setting the inventory card height to always match the events card. Another potential solution is to always have your events card at a defined height, introducing scroll for any information beyond that does not fit. Flexible and accommodating layouts usually provide the most effective long term solution.
```

Comment:
I think you're missing a "that" in the second sentence here: "Another potential solution is to always have your events card at a defined height, introducing scroll for any information beyond that **that** does not fit"?